modules/alarms_sns.tf

```diff
@@ -25,7 +25,7 @@ resource "aws_sns_topic_policy" "alarms_policy" {
           "SNS:DeleteTopic",
           "SNS:AddPermission"
         ],
-        "Resource": "aws_sns_topic.alarms_topic.arn",
+        "Resource": "${aws_sns_topic.alarms_topic.arn}",
```

Comment:
It looks like everywhere else you've _removed_ the `${..}` notation... why adding them here?

Reply:
Its because inside a text blob. ```policy = <<POLICY```.  I believe this is actually just a fix.

Reply:
Awesome. I'll approve, LGTM